actioncable/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "actioncable",
+  "version": "5.0.0-rc1",
+  "description": "WebSocket framework for Ruby on Rails.",
+  "main": "dist/actioncable.js",
+  "files": [
+    "dist/*.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rails/rails.git"
+  },
+  "keywords": [
+    "websockets",
+    "actioncable",
+    "rails"
+  ],
+  "author": "David Heinemeier Hansson <david@loudthinking.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/rails/rails/issues"
+  },
+  "homepage": "http://rubyonrails.org/",
+  "scripts": {}
+}
```

Comment:
How would publishing work then, if we set `"private": true`? `npm publish` would fail: `If you set "private": true in your package.json, then npm will refuse to publish it.`